src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -151,11 +154,11 @@ export class ESLNote extends ESLBaseElement {
     this._onBPChange();
   }
 
-  /** Revises the settings for the print version of note */
-  public updatePrintedQuery(): void {
-    memoize.clear(this, 'queryToPrint');
-    this.$$on(this._onPrintChange);
-    this._onPrintChange();
+  /** Revises the settings for displaying anchor inside of a note */
+  public updateAnchoredQuery(): void {
```

Comment:
```suggestion
  public updateAnchorQuery(): void {
```
as technically it's an anchor **'s** query